packages/experimental/Drawer/jest.config.js

```diff
@@ -0,0 +1,2 @@
+const { configureReactNativeJest } = require('@uifabricshared/build-native');
```

Comment:
I don't know if you need this file if you're not running any tests that depend on jest... (like snapshot tests or unit tests)